fiftyone/utils/coco.py

```diff
@@ -2157,9 +2157,19 @@ def _coco_segmentation_to_mask(segmentation, bbox, frame_size):
     width, height = frame_size
 
     if isinstance(segmentation, list):
+        # Filter out empty lists. For polygons of 4 points (1 pixel), duplicate
```

Comment:
This is undoing https://github.com/voxel51/fiftyone/commit/4658b6ecfd7747a40f30751e2936bc9b2984fe98 which is in `develop` but wasn't included in https://github.com/voxel51/fiftyone/pull/3595 because it only included merge commits.

I think we need to revisit the task of pulling changes from `develop` into `release/v0.22.1` before doing any merges in the other direction.

Reply:
Yeah, I had a 50/50 shot on this one - didn't do enough research and picked the wrong one.

Should be fixed now.